README.md

```diff
@@ -86,7 +93,44 @@ If you're making a package within a community that has an existing package templ
 
 ## Notes for developers
 
+### Contributing
+
+The [website] and source pages are publicly available, but contributions are restricted to members and associate members of ARC.
```

Comment:
How come? I would welcome someone's cool suggestion if we agree with it?

Reply:
Just copying over what was written in the website's README.

I assume it's so that it's clear it's just ARC recommendations, but I don't remember if we consciously agreed that back in hackday 1.

No strong opinion. 

How about something like:
```
The [website] and source pages are open to contributions but must be reviewed by a member or associate member of ARC.
We might be slow to approve new tool suggestions (since we'll probably want to discuss them first).
Bug reports (and fixes) are very welcome from anyone!
```

...?

Reply:
Yeah sounds good